services/app-api/src/domain-models/contractAndRates/baseContractRateTypes.ts

```diff
@@ -11,6 +11,64 @@ import {
 } from './revisionTypes'
 import { statusSchema } from './statusType'
 
+const divisionType = z.union([
+    z.literal('DMCO'),
+    z.literal('DMCP'),
+    z.literal('OACT'),
+])
+const stateType = z.object({
+    stateCode: z.string(),
+    name: z.string(),
+})
+const cmsUserType = z.object({
+    id: z.string().uuid(),
+    role: z.string(),
+    email: z.string(),
+    givenName: z.string(),
+    familyName: z.string(),
+    stateAssignments: z.array(stateType),
+    divisionAssignment: divisionType.optional()
+})
+const document = z.object({
+    name: z.string(),
+    s3URL: z.string(),
+    downloadURL: z.string().optional()
+})
+const stateUserType = z.object({
+    id: z.string().uuid(),
+    role: z.literal('STATE_USER'),
+    email: z.string(),
+    stateCode: z.string(),
+    givenName: z.string(),
+    familyName: z.string(),
+})
+const questionResponseType = z.object({
```

Comment:
Since we're redefining all these types here in Zod, let's get rid of our extant QuestionType s and export these generated zod types instead

Reply:
I think we need a plan for what to do with the types here generally. Also defined in this file is StateType and UserType CMSUserType,  all types already and now being moved into zod.. 

Maybe we need a ticket in ContractApi epic to clean this up after this ticket? I would expect also each of these to be in their own file not all living within the base contract type since they can be used across domains.

Reply:
So long as these zod types match the current explicit types, I think the fix here should be just defining the zod types in the same files where we have the explicit types instead of here, and then exporting the generated-from-zod types, deleting our explicit types. I think that's worth doing in this PR unless it hits some weird recursive type issue.